src/responder/pam/pamsrv_cmd.c

```diff
@@ -1072,6 +1072,11 @@ static void pam_reply(struct pam_auth_req *preq)
                     pam_account_expired_message);
     }
 
+    if (pd->pam_status == PAM_MAXTRIES) {
+        pd->pam_status = PAM_AUTH_ERR;
+        inform_user(pd, SSS_PAM_USER_INFO_PIN_LOCKED, "PIN locked");
```

Comment:
@sumit-bose, does this message need a localization?

Reply:
Another this is, probably it's not worth providing this string from `sssd_pam` to `sss_client` at all. It could be hard-coded in `sss_client`...

Reply:
Hi,

sorry for the delay. Yes, I think it is a good idea to have a localization for this string and as a consequence it would be better to just send the type and let pam_sss generate the message, in the locale of the calling environment, similar to `SSS_PAM_USER_INFO_OTP_CHPASS` and `user_info_otp_chpass()`.

bye,
Sumit

Reply:
Done.